build fails on OS X

When I run `make build` on a fresh checkout under OS X (Mavericks w/Oracle Java 1.8) I get:

```
atlas$ make
project/sbt clean test checkLicenseHeaders
java.io.IOException: No such file or directory
    at java.io.UnixFileSystem.createFileExclusively(Native Method)
    at java.io.File.createNewFile(File.java:1006)
    at xsbt.boot.Locks$.apply0(Locks.scala:34)
    at xsbt.boot.Locks$.apply(Locks.scala:28)
    at xsbt.boot.Launch.locked(Launch.scala:238)
    at xsbt.boot.Launch.app(Launch.scala:147)
    at xsbt.boot.Launch.app(Launch.scala:145)
    at xsbt.boot.Launch$.run(Launch.scala:102)
    at xsbt.boot.Launch$$anonfun$apply$1.apply(Launch.scala:35)
    at xsbt.boot.Launch$.launch(Launch.scala:117)
    at xsbt.boot.Launch$.apply(Launch.scala:18)
    at xsbt.boot.Boot$.runImpl(Boot.scala:41)
    at xsbt.boot.Boot$.main(Boot.scala:17)
    at xsbt.boot.Boot.main(Boot.scala)
Error during sbt execution: java.io.IOException: No such file or directory
make: *** [build] Error 1
```

--- a/project/sbt
+++ b/project/sbt
@@ -12,7 +12,7 @@ java \
   -Xmx2g \
   -XX:ReservedCodeCacheSize=128m \
   -XX:+UseCodeCacheFlushing \
-  -Dsbt.boot.directory=$WORKSPACE/.sbt \
-  -Dsbt.ivy.home=$WORKSPACE/.ivy2 \
+  -Dsbt.boot.directory=${WORKSPACE:-$HOME}/.sbt \
+  -Dsbt.ivy.home=${WORKSPACE:-$HOME}/.ivy2 \
   $OPTIONS \
   -jar `dirname $0`/sbt-launch.jar "$@"

